src/zenml/zen_stores/base_zen_store.py

```diff
@@ -142,6 +142,27 @@ def get_store_class(store_type: StoreType) -> Type["BaseZenStore"]:
                 f"`{store_type.value}`."
             )
 
+    @staticmethod
+    def get_store_type(url: str) -> Optional[StoreType]:
+        """Returns the store type associated with a URL schema.
+
+        Args:
+            url: The store URL.
+
+        Returns:
+            The store type associated with the supplied URL schema or None if
+            the store type cannot be determined.
+        """
+        from zenml.zen_stores.rest_zen_store import RestZenStoreConfiguration
+        from zenml.zen_stores.sql_zen_store import SqlZenStoreConfiguration
+
+        if SqlZenStoreConfiguration.supports_url_scheme(url):
+            return StoreType.SQL
+        elif RestZenStoreConfiguration.supports_url_scheme(url):
+            return StoreType.REST
+
+        return None
```

Comment:
When would this happen? Intuitively, I feel like this should raise an error instead of the `None` return in such cases

Reply:
Good point. Done

Reply:
Done